lib/pipefy_message/providers/aws_client/sns_broker.rb

```diff
@@ -28,23 +28,33 @@ def initialize(_opts = {})
         ##
         # Publishes a message with the given payload to the SNS topic
         # with topic_name.
-        def publish(payload, topic_name, context = nil)
+        def publish(payload, topic_name, context = nil, correlation_id = nil)
```

Comment:
I've been thinking about this `context` and I feel it's possibly a little confusing, given that we already have a `context: async_processing` in our logs (this would be nested one level deeper than that - `{ ... context: async_processing, ... message: { ... context: ... } } `). Should we rename this `message_context`, to highlight it related to where the event came from? Or perhaps even drop it altogether, now that we have correlation and event IDs (though this could potentially be more human-readable; `context: formula_fields, correlation_id: ...`)? WDYT?

Reply:
[See here](https://github.com/pipefy/pipefy_message/pull/33/files#r934832526)